### ISSUE TITLE
turnOnWhenOff not working 

### ISSUE THREAD

**Issue by josemunozmatos** [newt10/homebridge-broadlink-rm-enhanced#22]
When my AC is off and I turn it on, is sending a temp hex code instead of the On hex code.
Is there a way to send the On hex then the temp hex?

I tried "turnOnWhenOff": true but is not working. Does not send ON hex

Heres is my config for reference:
{
"name": "Living Room Cooler",
"type": "heater-cooler",
"host": "192.168.68.100",
"minTemperature": 68,
"maxTemperature": 76,
"coolingThresholdTemperature": 72,
"heatingThresholdTemperature": 76,
"temperatureUnits": "F",
"defaultNowTemperature": 74,
"turnOnWhenOff": true,
"debug": true,
"data": {
"cool": {
"on": "on_HEX_CODE",
"off": "off_HEX_CODE",
"swingToggle": "swing_HEX_CODE",
"temperatureCodes": {
"68": "68_HEX_CODE",
"69": "69_HEX_CODE",
"70": "70_HEX_CODE",
"71": "71_HEX_CODE",
"72": "72_HEX_CODE",
"73": "73_HEX_CODE",
"74": "74_HEX_CODE",
"75": "75_HEX_CODE",
"76": "76_HEX_CODE"
}
}
}
},

Log:

[6/28/2021, 18:25:07] [BroadlinkRM-Enhanced] Living Room Cooler setCurrentHeaterCoolerState: 0
[6/28/2021, 18:25:07] [BroadlinkRM-Enhanced] Living Room Cooler setCurrentHeaterCoolerState: already 0 (no data sent - B)
[6/28/2021, 18:25:10] [BroadlinkRM-Enhanced] Living Room Cooler setActive: 1
[6/28/2021, 18:25:10] [BroadlinkRM-Enhanced] Looking up temperature hex codes for 75
[6/28/2021, 18:25:10] [BroadlinkRM-Enhanced]    Sending hex codes for temperature 75
[6/28/2021, 18:25:10] [BroadlinkRM-Enhanced] Living Room Cooler sendHex (192.168.68.100; 42:f7:c8:83:61:2e) 2600600100011e9513371238121311151015111313131238111411381238121411141213111511141138123712141213111412141213121312131237111411141214121410141337121410141313101411151212131411141114111411141014111512131116101311000106121312131237131311381214113912371238131310391214111411141115101412141114111410151015111412131214121212141114111410141312121511141114111411141113111512131214111313131114101511151013131312131213121412121214111312141213121413111313121212141113133712131239121312000104131212141114121311141115111412131312113812141113121313141114121311131313111411141213111510151114121311141212131410151014131312141113121312131237131312141015121311131312131311131214121411141213121312371313123711141313111412

**Comment by newt10** [comment]
I don't believe this is a supported feature in this accessory at the moment. Can you share more details on your device and how the remote for it works currently?

1. Does your remote allow you to set a specific temperature?
2. Does you AC turn on to the last temperature setting when you turn it on?
3. If you press the temperature buttons on your AC when it is powered off, does it turn it on or is there no effect?

Please share the details on these questions so I can help debug it. Thanks.

**Comment by josemunozmatos** [comment]
Hello, thanks for your response! Of course here is the info:

1. No, it only has temp-up/temp-down arrows
2. No, the turn on/off is a toggle. It turns on to the temp "binded" into the ON Hex so I recorded the ON hex at 74 it will always turn on at 74, and so on.
3. If I press the temp buttons while AC is off, it does nothing.

**Comment by newt10** [comment]
ok, let me work on an feature to fix this behavior. I should have something working next work for you to try. Would you be able to help beta test it?

**Comment by josemunozmatos** [comment]
Hi! Yess of course would love to help.
